lib/api/settings.js

```diff
@@ -0,0 +1,39 @@
+const Request = require('./request')
+
+class Settings extends Request {
+
+  constructor(httpClient){
+    super(httpClient)
+    this.httpClient = httpClient
```

Comment:
už to posíláš do parenta přes `super(httpClient)`, tak httpClienta nepřiřazuj v této třídě.

Reply:
vubec nevim kde se to tam vzalo, asi zase nejaky zmatek ve verzovani